src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,25 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+
+    private static final SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil() {
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Throwable e) {
+            System.err.println("Initial SessionFactory creation failed." + e);
+            throw new ExceptionInInitializerError(e);
+        }
```

Comment:
While catching `Throwable` is a common pattern in static initializers to prevent class loading if an error occurs, it's often better to catch more specific exceptions if possible. Also, consider logging the exception using a proper logging framework instead of `System.err.println` to provide more flexibility and control over the log output.